scripts/src/test/bash/integration-test-docker

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+source "$(dirname "${0}")"/test-functions
+doCommandTest docker
```

Comment:
Typically `docker` is already present inside github action environment.
Do we have any idea what happens if we run this on windows (or mac)?
IMHO a real installation of Rancher Desktop on Windows can never work via github action as it shows installation UI screens that a user has to interact with.